styleguide/source/_patterns/02-molecules/inline-block-copy.twig

```diff
@@ -1,8 +1,8 @@
 {% set heading, paragraph, button = inlineBlockCopy.heading, inlineBlockCopy.paragraph, inlineBlockCopy.button %}
 
-<div class="ama__inline-block-copy">
+<div class="ama__inline-block-copy col-sm-12 col-md-12 col-lg-4">
```

Comment:
`col-md-12` and `col-sm-12` are still setting this element to `width: 100%` and it doesn't appear that `col-lg-4` is having any effect at all. I would recommend handling the sizing directly within the `scss` for `ama__inline-block-copy`

Reply:
@labbydev perhaps I'm confused how this should be viewed on mobile devices but I thought we wanted the inline block to render at 100% on tablet and mobile and around 25% on desktop. If the grid sizing is not working then we will need to revisit it right away.